src/Controller/DocumentController.php

```diff
@@ -93,6 +91,7 @@ public function create(CaseEntity $case, Request $request): Response
 
                 // Set filename, document name, creator and case
                 $document->setFilename($newFilename);
+                $document->setPath($this->documentUploader->getUploadDocumentDirectory().'/'.$newFilename);
```

Comment:
I think the DocumentUploader should handle this. Where is actually stores files should be kept inside the uploader.

Reply:
Added `getFilepathFromProjectDirectory($filename)` method to `DocumentUploader`.